docs/source/reference/rest_api.rst

```diff
@@ -946,49 +995,246 @@ All of them must be followed by the operator ``=``.
+
+.. list-table:: Filter operators
+    :header-rows: 1
+
+    * - Operator
+      - Meaning
+      - Accepted value types
+    * - ``=``
+      - identity
+      - integers, strings, bool, datetime
+    * - ``>``
+      - greater than
+      - integers, strings, datetime
+    * - ``<``
+      - less than
+      - integers, strings, datetime
+    * - ``>=``
+      - greater than or equal to
+      - integers, strings, datetime
+    * - ``<=``
+      - less than or equal to
+      - integers, strings, datetime
+    * - ``=like=``
+      - pattern matching
+      - strings
+    * - ``=ilike=``
+      - case-insensitive pattern matching
+      - strings
+    * - ``=in=``
+      - identity with one element of a list
+      - integers, strings, datetime
+
+
+Pattern matching
+^^^^^^^^^^^^^^^^
 
-Regular filters
----------------
+The pattern matching operators ``=like=`` and ``=ilike=`` must be followed by the pattern definition, namely, a string where two characters assume special meaning:
+
+    1. ``%`` is used to replace an arbitrary sequence of characters, including no characters.
+    2. ``_`` is used to replace one or zero characters.
 
-Regular filters can be compounded (requiring all specified filters to apply):
+.. note:: When special characters are required verbatim, escape them by pre-pending a backslash ``\``.
+
+.. list-table:: Pattern matching with ``=like=`` and ``=ilike=``
+    :header-rows: 1
+
+    * - Filter
+      - Matches
+      - Doesn't match
+    * -  ``name=like="a%d_"``
+      -  "aiida"
+      -  "AiiDA"
+    * -  ``name=ilike="a%d_"``
+      -  "aiida", "AiiDA"
+      -
+    * -  ``name=like="a_d_"``
+      -
+      -  "aiida"
+    * -  ``name=like="aii%d_a"``
+      -  "aiida"
+      -
+    * -  ``uuid=like="cdfd48%"``
+      - "cdfd48f9-7ed2-4969-ba06-09c752b83d2"
+      -
+    * - ``description=like="This calculation is %\% useful"``
+      - "This calculation is 100% useful"
+      -
+
+Membership
+^^^^^^^^^^
+
+The membership operator ``=in=`` has to be followed by a comma-separated list of values of the same type.
+The condition is fulfilled if the column value of an object is an element of the list.
+
+Examples::
+
+    http://localhost:5000/api/v4/nodes?id=in=45,56,78
+    http://localhost:5000/api/v4/computers/?scheduler_type=in="slurm","pbs"
 
-    :string:
+Comparison
+^^^^^^^^^^^^^^^^^^^^
+
+The comparison operators ``<``, ``>``, ``<=``, ``>=`` assume natural ordering for integers, (case-insensitive) alphabetical ordering for strings, and chronological ordering for datetime values.
+
+Examples:
+
+    - ``http://localhost:5000/api/v4/nodes?id>578`` selects the nodes having an id larger than 578.
+    - ``http://localhost:5000/api/v4/users/?last_name<="m"`` selects only the users whose last name begins with a character in the range [a-m].
+
+
+
+
+Filter value types
+------------------
+
+Filter values should be specified as follows:
+
+.. list-table:: Regular filters
```

Comment:
Maybe sort this list alphabetically as well?

Reply:
done!